<launch>
    <rosparam file="$(find custom_dynamixel_pack)/config/hardware.yaml" command="load"/>
    <rosparam file="$(find custom_dynamixel_pack)/config/controllers.yaml" command="load"/>
    <node name="ROBOT_hardware_interface" pkg="custom_dynamixel_pack" type="custom_dynamixel_pack_node" output="screen"/>   
<node name="controller_spawner" pkg="controller_manager" type="spawner" respawn="false" output="screen" ns="/"
       args="
            /ROBOT/controller/state
            /ROBOT/controller/position/YOUR_ROBOT_JOINT
        "/>
</launch>
